corehq/motech/fhir/models.py

```diff
@@ -20,6 +20,7 @@
     as_value_source,
 )
 from corehq.motech.fhir import serializers  # noqa # pylint: disable=unused-import
+from corehq.motech.fhir import signals  # noqa # pylint: disable=unused-import
```

Comment:
This is the line that does the heavy lifting: Django imports all the models modules, so this import ensures that `create_fhir_repeat_records()` is connected to the `successful_form_received` signal.